packages/dds/tree/package.json

```diff
@@ -127,6 +151,34 @@
 		"ts2esm": "^1.1.0",
 		"typescript": "~5.1.6"
 	},
+	"fluidBuild": {
+		"tasks": {
+			"build:docs": [
+				"build:esnext"
+			],
+			"build:esnext": [
+				"^api",
+				"^build:esnext"
+			],
+			"ci:build:docs": [
+				"build:esnext"
+			],
+			"commonjs": {
```

Comment:
I don't think the dependency changes here and below are needed. Look like leftovers when there was an attempt to have `tsc` be for ESM build.